src/infrastructure/database/modules/sqlite/repositories/sqlite-status-list.repository.ts

```
@@ -0,0 +1,228 @@
+        options
+      });
+      throw new Error(`Failed to find status lists: ${error instanceof Error ? error.message : String(error)}`);
+    }
+  }
+
+  /**
+   * Updates a status list
+   */
+  async update(id: Shared.IRI, updates: Partial<Omit<StatusList, 'id' | 'createdAt'>>): Promise<StatusList | null> {
+    try {
+      const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)